packages/shopping/src/controllers/user-managemet.controller.ts

```diff
@@ -91,23 +94,12 @@ export class UserController {
     // ensure a valid email value and password value
     validateCredentials(_.pick(newUserRequest, ['email', 'password']));
 
-    // encrypt the password
-    const password = await this.passwordHasher.hashPassword(
-      newUserRequest.password,
-    );
-
     try {
-      // create the new user
-      const savedUser = await this.userRepository.create(
-        _.omit(newUserRequest, 'password'),
+      const userManagementService = new UserManagementService(
```

Comment:
I think a better way is to inject the user management service here, instead of creating a new one.